### DISCUSSION TITLE
Help Improving Accuracy of Vessels Segmentation

### DISCUSSION THREAD

**Discussion by soumen02** [Project-MONAI/MONAI#6215] [Q&A]
Hi everyone,

I'm using **MONAI Label's deepedit app** to segment vessels in a dataset of 21 patients. I trained the **Dynunet** network for **500 iterations** and also tried increasing the number of epochs, but I'm not seeing a significant improvement in the results.
As you can see in the image below, the actual label is in red and the predicted label is in green.
I would appreciate any suggestions or feedback on how to improve the segmentation results. 

Thank you in advance!

![image](https://user-images.githubusercontent.com/38569809/226727912-5c7dd1ac-439d-4da0-b6ee-8116f3311f75.png)

**Comment by Nic-Ma** [comment]
Hi @tangy5 ,

Could you please help share some comments about this ticket?

Thanks in advance.

**Reply by tangy5** [reply]
Happy to help, I think we already had a discussion in the MONAI Slack channel. Let's catch there. 
Thank you!
